packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -955,7 +955,7 @@ export class Project extends ProjectBase {
     }));
 
     const policy = new iam.Policy(this, 'PolicyDocument', {
-      policyName: 'CodeBuildEC2Policy',
+      policyName: this.node.uniqueId + 'CodeBuildEC2Policy',
```

Comment:
Can you just remove `policyName` property from this line? This way, the CDK will use the generated ID of the `Policy` construct as the `policyName`.

Reply:
Of course, your suggestion seems a better way!